init/cmdline.go

```diff
@@ -122,7 +219,7 @@ func parseParams(params string) error {
 			if len(parts) != 2 {
 				return fmt.Errorf("invalid rd.luks.name kernel parameter %s, expected format rd.luks.name=<UUID>=<name>", value)
 			}
-			uuid, err := parseUUID(stripQuotes(parts[0]))
```

Comment:
Can the `stripQuotes()` function be removed now?

Reply:
It does look like it, everything it was used for came from the cmdline parser and it cleans them up automatically now